mailpoet/mailpoet
# Use wordpress i 18 n in all of our codebase [MAILPOET-4908]

## Description

This PR uses @wordpress/i18n package in the js/ts code related to Newsletters and the common code ( the directory mailpoet/assets/js/src/common ). 

## Code review notes
There will be a follow up ticket to take care of rest of the codebase.

## QA notes

Please switch your site's language to a language that has a good coverage and check if the plugin remains translated on different pages

## Linked tickets

[MAILPOET-4908]

## After-merge notes

Please run the following commands to keep the translations current for the release to work fine:
- ` ./do translations:build `
- ` ./do translations:push `
- Run `./do release:version-get-next` to get the next version to use in the next step
- `./do translations:prepare-language-packs {version from previous step}`


[MAILPOET-4908]: https://mailpoet.atlassian.net/browse/MAILPOET-4908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
